lib/ansible/module_utils/k8s/raw.py

```diff
@@ -78,6 +78,7 @@ def argspec(self):
         argument_spec.update(copy.deepcopy(AUTH_ARG_SPEC))
         argument_spec['merge_type'] = dict(type='list', choices=['json', 'merge', 'strategic-merge'])
         argument_spec['wait'] = dict(type='bool', default=False)
+        argument_spec['wait_sleep'] = dict(type='int', default=1)
```

Comment:
I really don't want to be calling the API this often. Can we make this 6 by default (as that's what it was before the change)

Reply:
For rounding I changed the default value to 5 seconds. Is it ok?

Reply:
Sure, why not :)